Add Predictions to API

LittleTealeaf opened the pull request:
This feels a bit rough. Would it be better if we made it more *relative* to the current value? Perhaps the predicted spots is the current spots +/- some random value?

Currently it just returns a random number

LittleTealeaf committed:
Add Predictions to API

LittleTealeaf committed:
Added javadocs  to new methods

kevin-rodri commented:
Let's make it a bit more realistic instead of returning a random number. I think would be better off having the getPredictedOpenSpots method return a value from currentOpenSpots +/- a random value. That way, it's in sync with the getPredictedClosedSpots method.

kevin-rodri commented:
whoops, I didn't mean to accept the pull request yet...

LittleTealeaf committed:
Added relativeness to getPredictedOpenSpots()

Results of the getPredictedOpenSpots() is now relative to the current
number of available spots (meaning that the result will typically be
around that count)
